Add visual relationships component to unified experience (#868)
sourcegraph/handbook: content/company/strategy/code-graph/index.md

### Deliver a unified experience

> Our product solves real problems today, but investing in the polish, user experience, and cross-feature functionality within our product teams to make the experience lovable and help us deliver solutions that are more than the sum of the parts. As an example flow: Code Insights can lead you to a more detailed Precise Code Intelligence search, from which you might launch a more advanced semantic analysis; from here, you might use Batch Changes to remediate a problem and then document everything you've done in a Search Notebook, monitoring the progress as the fix is rolled out in Code Insights. Another example might be learning about a library in API docs and exploring how it is used over time using Code Insights.

Reviewer: > Our product solves real problems today, but investing in the polish, user experience, and cross-feature functionality within our product teams to make the experience lovable and help us deliver solutions that are more than the sum of the parts.

This sentence reads difficult? Or am I just not understanding it properly?

Author: No you're right @jeanduplessis . Better?

```suggestion
Our product solves real problems today, but by investing in polish, user experience, and integrated functionality we can make the experience more lovable and deliver solutions that are "more than the sum of their parts." As an example flow: Code Insights can lead you to a more detailed Precise Code Intelligence search, from which you might launch a more advanced semantic analysis; from here, you might use Batch Changes to remediate a problem and then document everything you've done in a Search Notebook, monitoring the progress as the fix is rolled out in Code Insights. Another example might be learning about a library in API docs and exploring how it is used over time using Code Insights.
```

Reviewer: 👌  much better 🙇
